data/tlg0086/tlg034/__cts__.xml

```diff
@@ -1,22 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0086" projid="greekLit:tlg034" urn="urn:cts:greekLit:tlg0086.tlg034" xml:lang="grc">
             
-   <ti:title xml:lang="eng">Ars Poetica</ti:title>
+<ti:title xml:lang="eng">Poetics</ti:title>
             
-   <ti:edition projid="greekLit:digicorpus-grc1" urn="urn:cts:greekLit:tlg0086.tlg034.digicorpus-grc1" workUrn="urn:cts:greekLit:tlg0086.tlg034" xml:lang="grc">
-                
-      <ti:label xml:lang="lat">Ars Poetica</ti:label>
-                
-      <ti:description xml:lang="mul">Aristotelis Opera, ed. Immanuel Bekker. Oxford, Oxford
-                    University Press, 1837. </ti:description>
-                
-   </ti:edition>
+<ti:edition projid="greekLit:digicorpus-grc1" urn="urn:cts:greekLit:tlg0086.tlg034.digicorpus-grc2" workUrn="urn:cts:greekLit:tlg0086.tlg034" xml:lang="grc">
+<ti:label xml:lang="grc">Περὶ ποιητικῆς</ti:label>
+<ti:description xml:lang="mul">Aristotle. Aristotelis Opera, Vol. 11. Bekker, Immanuel, editor. Oxford: Oxford University Press, 1837. </ti:description>
```

Comment:
My only thought here is that for many cts descriptions where the author's name is implied in the title I thought we had been eliminating author names and would just have Aristotelis Opera, Vol. 11..etc.

Reply:
Generally not when the author name is in Latin (and not even in the nominative case).